src/views/about/About.js

```diff
@@ -70,42 +58,91 @@ const About = () => {
                                 To get started, visit the <Link to='/data'>Data</Link> page to view all the sunshine list data. Afterwards, you can visit the <Link to='/visualize'>Visualize</Link> page to dive deep into the data!
                             </p>
                         </Panel>
-                        <Panel header="How was Sundial made?" key="2">
+                        <Panel header="1IA" key="2">
                             <img class="sizeB" src={image}></img>
+                            <br/>
                             <p>
                                 Sundial was developed by a small effective team of prodigy programmers who formed a team called 1IA.<br/>Which is an acronym for One iteration Agile. This used this name because it best describes their design process.
```

Comment:
```suggestion
                                Sundial was developed by a small effective group of prodigy programmers who formed a team called 1IA.<br/>Which is an acronym for One iteration Agile. They used this name because it best describes their design process.
```